Makefile

```diff
@@ -188,7 +188,7 @@ go.mod: $(shell find . -name '*.go')
 go.sum: go.mod
 
 sdk: $(shell mkdir -p sdk)
-sdk: sdk/python sdk/nodejs sdk/java sdk/python sdk/go sdk/dotnet
+sdk: sdk/python sdk/nodejs sdk/java sdk/go sdk/dotnet
```

Comment:
This is unrelated to the main change, but I noticed that `sdk/python` is duplicated so I removed one of those.